### PR TITLE
Implement cloudinary for workspace

### PR DESCRIPTION
**Pull request by ericblandet**
please have a look and comment
- added .env + gitignore
- added cloudinary gem
- added credential cloudinary in .env
- added has many attached in Model workspace
- added attached images in seed

### CODE REVIEW

File: app/controllers/workspaces_controller.rb [hunk 1]
```diff
@@ -28,6 +28,6 @@ def set_workspace
   end
 
   def workspace_params
-    params.require(:workspace).permit(:name, :description, :address, :capacity, :daily_route)
+    params.require(:workspace).permit(:name, :description, :address, :capacity, :daily_route, :photo)
```
Comment (Simon-Busch) [inline app/controllers/workspaces_controller.rb]: photo S

**Review by Simon-Busch — CHANGES_REQUESTED** [review]
change photo to photos

File: app/controllers/workspaces_controller.rb (hunk 1 above)
Comment (ericblandet) [reply app/controllers/workspaces_controller.rb]: ok

**Review by Simon-Busch — APPROVED** [review]
ok for me

**Pull request merged**